Adapt, apply or drop ESR60 patches for ESR78. JB#55228 OMP#JOLLA-298

llewelld opened the pull request:
These changes adapt, apply or drop patches that were applied to ESR60 in the context of ESR78.

rainemak commented on rpm/0052-sailfishos-gecko-Make-TextureImageEGL-hold-a-referen.patch:
```diff
@@ -1,4 +1,4 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From a642f9e2ebd9a862de0cee7e9e77d33ab9547ef7 Mon Sep 17 00:00:00 2001
```
Should be formatted like...

> git format-patch --zero-commit  -N sha1

That said just zeroing this would be enough.

llewelld replied:
Thanks; I've zeroed it out.

rainemak commented on rpm/0053-sailfishos-gecko-Make-fullscreen-enabling-work-as-us.patch:
```diff
@@ -1,8 +1,8 @@
 Date: Wed, 19 Aug 2020 14:27:21 +0300
-Subject: [PATCH] [sailfishos][gecko] Make fullscreen enabling work as used to
- with pref full-screen-api.content-only. Fixes JB#44129
+Subject: [PATCH 1/1] [sailfishos][gecko] Make fullscreen enabling work as used
```
git format-patch --zero-commit -N sha1

llewelld replied:
Thanks again; I've changed this one too.

llewelld commented:
Rebased, and now ready for review.

llewelld commented:
Just for info, this may be easier to review by taking the separate commits individually.

rainemak reviewed (approved):
You need to rebase this again and adjust patch numbers. Approving already, please merge once you get this rebased. Surely, if you like I can give another look after rebasing.

llewelld commented:
Rebased and adjusted the patch numbers. It's not got through a full rebuild yet, but the patches applied just fine after the renumbering, so I think it should be fine.

llewelld commented:
Thanks for holding back #69 Raine. To avoid blocking more things, I'll merge this now; if you and Andrey were happy with it before I think it should be okay.
